src/emmetHelper.ts

```diff
@@ -169,8 +169,8 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
-function removeTabStops(expandedWord: string): string {
-	return expandedWord.replace(/\$\{\d+\}/g, '').replace(/\$\{\d+:([^\}]+)\}/g, '$1');
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_') + '|';
```

Comment:
Okay, I added the last cursor, because after each inserted snippet, the cursor is behind it, that was my reason.

Tbh, I didn't looked quite deeply into the code, but only looked for the changes I did last time.

Will fix this as soon as I got time.